src/shogun2-core/src/main/java/de/terrestris/shogun2/hibernate/SimpleJsonbConverter.java

```diff
@@ -0,0 +1,51 @@
+package de.terrestris.shogun2.hibernate;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.log4j.Logger;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * A JPA converter that can be used to convert jsonb fields into a map. Currently only works with 'simple' json
+ * content (only one level objects like {"name": "peter", "someprop": 345}).
+ *
+ * Please note that you'll need to switch to pgjdbc-ng from https://github.com/impossibl/pgjdbc-ng instead of using the
+ * standard postgres driver.
+ *
+ * Please also note that if you want optimal performance you'll need to add an index manually, hibernate can only create
+ * btree indexes.
+ */
+@Converter(autoApply = true)
+public class SimpleJsonbConverter implements AttributeConverter<Map<String, String>, String> {
+
+    private static final Logger LOG = Logger.getLogger(SimpleJsonbConverter.class);
+
+    @Override
+    public String convertToDatabaseColumn(Map<String, String> attribute) {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
I don't think that this will make a difference, but maybe we should use an instance of `Shogun2JsonObjectMapper` here?

https://github.com/terrestris/shogun2/blob/master/src/shogun2-core/src/main/java/de/terrestris/shogun2/util/json/Shogun2JsonObjectMapper.java

Reply:
I wasn't aware of this one, thanks! I think it might still make a difference, I'm sure we'll be storing timestamps in there, so I changed it.